mautic/mautic
# Rewording notifications

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [y]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | /



#### Description:

The Notification permissions are confusing as they show only “Notifications - User has access to” section. Users may be confused about what notifications are they configuring. It should say "Web Notifications" as it is used in the menu:

![Screen Shot 2022-06-15 at 09 36 25](https://user-images.githubusercontent.com/1235442/173771344-919d9720-020e-4783-9d18-4873bbdfacf2.png)

This PR is changing this:
![Screen Shot 2022-06-15 at 10 08 20](https://user-images.githubusercontent.com/1235442/173777773-3cc6f7d2-3456-4639-a0b4-ee11eaa271f5.png)




Into this:
![Screen Shot 2022-06-15 at 10 10 56](https://user-images.githubusercontent.com/1235442/173776951-cfc75e98-6b67-41cf-808b-8fe217777789.png)


Also, for some reason the notification permission translations lived in the WebhookBundle. I'm moving them to the NotificationBundle where they belong. I [added them to the wrong bundle](https://github.com/acquia/mc-cs/commit/bae4954352f95b0bba4c2d26ecafe2b4931e4b95) back in 2016 for unknown reason. What an ignorant I was back then :) 

Note: The menu items are visible only when the OneSignal plugin is enabled.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create new permissions and notice the translation changes from the screenshots. Make sure you are using the EN - US translations which are the default ones.

#### Other areas of Mautic that may be affected by the change:
1. Just translations

#### List deprecations along with the new alternative:
1. None


[//]: # ( As applicable: )
#### List of areas covered by the unit and/or functional tests:
1. No tests are necessary for wording changes
